### ISSUE TITLE
Mobile application 

### ISSUE THREAD

**Issue by dharmajmrd** [changeweb/Unifiedtransform#159]
can we have mobile version of this application. It will be useful for Teachers and parents to make use of features like Attendance, Homework, Marks, Events etc.

**Comment by changeweb** [comment]
We can build mobile version using React Native framework. But I'm a bit busy with my other works.

If you can contribute, it will be great.

**Comment by mykeels** [comment]
For a mobile application, the back-end would need to expose some form of API.

**Comment by changeweb** [comment]
I created a resources folder earlier but it and the api.php file need to be updated.

**Comment by dharmajmrd** [comment]
> We can build mobile version using React Native framework. But I'm a bit busy with my other works.
> 
> If you can contribute, it will be great.

yes. I have some exposure on Android development. In fact i have built a small android application with small features like taking attendance through Mobile application, Homework, School events, Fees history etc.. But i need some help on back end API exposure..

**Comment by changeweb** [comment]
I will work on API whenever possible. I hope it will be ready within next month.

**Comment by adaojunior** [comment]
Have you thought about building a PWA instead of a React Native app? 

I have experience with react/angular, and I'm currently studying project management. Let me know if any of those skills may be helpful in this project.
